### DISCUSSION TITLE
Is it possible to not use the Lite UI when launch KoboldCpp?

### DISCUSSION THREAD

**Discussion by kmcbest** [LostRuins/koboldcpp#145] [General]
By default KoboldCpp.exe launches with the Kobold Lite UI. However, many tutorial videos are using another UI which I think is the "full" UI, like this:
![image](https://user-images.githubusercontent.com/689023/236590966-4ccbb47f-fff1-4686-9984-ee88ede73f4e.png)
![image](https://user-images.githubusercontent.com/689023/236590995-ec0bbac7-eb84-4a60-93d9-ff2688b26a01.png)

Even on KoboldCpp's Usage section it was said "To run, execute koboldcpp.exe or drag and drop your quantized ggml_model.bin file onto the .exe, and then **connect with Kobold or Kobold Lite**. " But how can I connect with Kobold but not Lite?

**Comment by LostRuins** [comment]
You have to download the full kobold client from this repo, follow the instructions (about 10gb installed):

https://github.com/henk717/KoboldAI 

After installation, start both programs. Then you can select "AI" -> Online Services -> KoboldAI API

![image](https://user-images.githubusercontent.com/39025047/236595775-2ec1988a-6b79-4251-bb21-1870d15a5d29.png)

and use `http://localhost:5001` as the target

**Reply by kmcbest** [reply]
OK so it is not feasible for KoboldCpp to use the full UI? I really want to save 10GB of disk space because CPP is so lightweight.

**Reply by LostRuins** [reply]
Nope, the full UI is only available with Kobold United

**Reply by henk717** [reply]
For context, our normal UI's are very heavily tied into the main KoboldAI and without a KoboldAI server they can't do anything. While Lite is a recreation entirely in Javascript.
